src/Compilers/CSharp/Test/Symbol/Symbols/ModuleInitializersTests.Accessibility.cs

```diff
@@ -0,0 +1,392 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
+{
+    public sealed partial class ModuleInitializersTests
```

Comment:
>public [](start = 4, length = 6)

[CompilerTrait(CompilerFeature.ModuleInitializers)]? also check other new files.

Reply:
Even though it was already applied to the class?

Reply:
> Even though it was already applied to the class?

Probably not, I wasn't sure if the type is shared across multiple files and whether the attribute is applied to other partials.

---
In reply to: [418174679](https://github.com/dotnet/roslyn/pull/43301#discussion_r418174679) [](ancestors = 418174679)